flink-streaming-connectors/flink-connector-kafka/src/main/java/org/apache/flink/streaming/connectors/kafka/FlinkKafkaConsumer.java

```diff
@@ -290,9 +291,6 @@ public FlinkKafkaConsumer(String topic, DeserializationSchema<T> valueDeserializ
 			}
 		}
 		LOG.info("Topic {} has {} partitions", topic, partitions.length);
-
-		// make sure that we take care of the committing
-		props.setProperty("enable.auto.commit", "false");
```

Comment:
Is this not necessary, because the low-level consumer does not respect it anyways?